### ISSUE TITLE
Add languages

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#2014]
Add the following languages:
Arabic
Chinese
Japanese
Hindi
Swahili
Amharic
Coptic
Zulu

Alternatively, perhaps preferably, we could already add an extensive list of languages, but simplify their choice with a suggestion-based dropdown (you start writing and the list limits to the hits, often used in online apps exactly for lists of languages or countries). Of course, keeping all existing ones, and also the list will not of course be complete because somebody will want variants (Egyptian vs. Moroccan Arabic), historical languages (Mycenian, Dalmatian) or old stages of existing languages (as we have Middle English). But we need a tool which already in the default settings is more global.

**Comment by davidzbiral** [comment]
EDIT: the following is already implemented:
Old Church Slavonic
Old French
Old Italian
And relabel (do not add but relabel) Occitan to Old Occitan.

**Comment by Ptrhnk** [comment]
@davidzbiral So should we add the complete list of languages from the wikipedia link?

**Comment by davidzbiral** [comment]
Yes please, the good suggester will help. But also of course keeping our existing ones, esp. for old variants of languages.

**Comment by Ptrhnk** [comment]
@davidzbiral 

> the good suggester will help

First thing that comes to mind is to match ISO codes when typing, what do you think?

And then the questions are:
- Do you want to for example prioritize (rank) basic languages in results?
- And for opening the dropdown without typing - what about visually split common languages of your choice from the rest (or for example the languages used until now in InkVisitor separate from the new ISO additions)?

Together there will be 489 languages FYI.

**Comment by davidzbiral** [comment]
@Ptrhnk I suggest to implement this as part of the current issue #2025 (multichoice), and then, if any lang is set as my working language, then yes, show it as first in the list, both plain list with no user input, and list filtered by already user's input text, but other than this, same approach as so far. I think this solves your questions.
